src/tests/test_crypto.py

```diff
@@ -73,7 +73,8 @@ class TestCrypto(RIPEMD160TestCase, unittest.TestCase):
     """RIPEMD160 test case for Crypto"""
     @staticmethod
     def _hashdigest(data):
-        return RIPEMD.RIPEMD160Hash(data).digest()
+        from pybitmessage.fallback import RIPEMD160Hash
+        return RIPEMD160Hash(data).digest()
```

Comment:
You didn't get it. This part of the test should use `pycrypto` or `pycryptodome`, not fallback. Because it's the test for fallback itself.

Reply:
Comment stating this would've been helpful.